packages/nx/src/command-line/migrate/migrate.ts

```diff
@@ -1588,7 +1588,7 @@ function commitChanges(commitMessage: string): string | null {
   return getLatestCommitSha();
 }
 
-function getLatestCommitSha(): string | null {
+export function getLatestCommitSha(): string | null {
```

Comment:
I don't think this needs to be exported?

Reply:
oooops!